CHANGELOG.md

```diff
@@ -1,5 +1,413 @@
 # Changelog
 
+## [1.15.0] (2025-01-09)
+
+[1.15.0]: https://github.com/crystal-lang/crystal/releases/1.15.0
+
+### Breaking changes
+
+#### lang
+
+- Allow constants to start with non-ascii uppercase and titlecase ([#15148], thanks @nanobowers)
```

Comment:
What makes this a breaking change?

Reply:
Until now, identifiers starting with non-ascii uppercase and titlecase characters were considered variable names.
For example, the following code is considered valid in 1.14 but errors in 1.15:
```cr
Á = 1
Á = 2 # Error: already initialized constant Á
```